app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -32,22 +32,25 @@
 				"bureau_de_change",	"money_transfer", "post_office", "library",
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
-				"doctors", "clinic", "pharmacy", "veterinary" };
+				"doctors", "clinic", "pharmacy", "veterinary",
+				"place_of_worship", "school", "townhall"};
```

Comment:
Other than i.e. universities, schools can not be considered public in the sense that anybody may simply enter these places. In this sense, it is like a place of work (office, industrial landuse, ...). This is both relevant because the app should not ask the surveyor about private places (= places the surveyor may not be allowed to enter) and because the wheelchair-tag on a private place is not useful.